src/transforms/dedupe.rs

```diff
@@ -197,7 +198,7 @@ type TypeId = u8;
 #[derive(PartialEq, Eq, Hash)]
 enum CacheEntry {
     Match(Vec<Option<(TypeId, Bytes)>>),
-    Ignore(Vec<(String, TypeId, Bytes)>),
+    Ignore(Vec<(ConfigTargetPath, TypeId, Bytes)>),
```

Comment:
```suggestion
    Ignore(Vec<(OwnedTargetPath, TypeId, Bytes)>),
```
This isn't used in a `Config` struct, so it doesn't need the config path type.